ui/components/app/asset-list/asset-list.js

```diff
@@ -102,13 +102,12 @@ const AssetList = ({ onClickAsset }) => {
         )}
       <Box marginTop={detectedTokens.length > 0 ? 0 : 4}>
         <Box justifyContent={JustifyContent.center}>
-          <Typography
+          <Text
             color={Color.textAlternative}
-            variant={TypographyVariant.H6}
-            fontWeight={FONT_WEIGHT.NORMAL}
+            variant={TextVariant.bodySm}
```

Comment:
Ah oversight on my part we need to add html tags here so that some e2e tests pass. I've updated the description of the issue if you'd like to look.

```suggestion
            variant={TextVariant.bodySm}
            as="h6"
```